.golangci.yaml

```diff
@@ -1,18 +1,24 @@
-run:
-  # timeout for analysis, e.g. 30s, 5m, default is 1m
-  timeout: 90s
-
-linters-settings:
-  gocyclo:
-    # minimal code complexity to report, 30 by default (but we recommend 10-20)
-    min-complexity: 25
-  depguard:
-    rules:
-      main:
-        deny:
-          - pkg: "golang.org/x/net/context"
-          - pkg: "github.com/gogo/protobuf/proto"
-
-issues:
-  # Don't turn off any checks by default. We can do this explicitly if needed.
-  exclude-use-default: false
+version: "2"
+linters:
+  settings:
+    depguard:
+      rules:
+        main:
+          deny:
+            - pkg: golang.org/x/net/context
+            - pkg: github.com/gogo/protobuf/proto
+    gocyclo:
+      min-complexity: 25
+  exclusions:
+    generated: lax
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
+formatters:
+  exclusions:
+    generated: lax
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
These paths don't exist in this repo.

Reply:
I imagine it's just being explicit about defaults (or attempting to preserve defaults from earlier version?)